packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/languages/perl/changeArg4.yml

```diff
@@ -0,0 +1,22 @@
+languageId: perl
+command:
+  version: 4
+  spokenForm: change arg
+  action: {name: clearAndSetSelection}
+  targets:
+    - type: primitive
+      modifiers:
+        - type: containingScope
+          scopeType: {type: argumentOrParameter}
+  usePrePhraseSnapshot: true
+initialState:
+  documentContents: some_funky_func( "and", "three", "args" )
+  selections:
+    - anchor: {line: 0, character: 16}
+      active: {line: 0, character: 16}
+  marks: {}
+finalState:
+  documentContents: some_funky_func(  )
```

Comment:
I thought this issue was fixed in latest tree-sitter?

Reply:
It was fixed in the tree-sitter, not in my code apparently. With the latest commit I'm now testing for an error here instead as my current implementation does not manage to find the first argument (like e.g. the JS one does), something which I've not yet been able to solve. Behaviour of latest code is improved overall though.

Reply:
update from meet-up: let's punt in getting this one to work; but see my other comment about whether we want to leave characterization test or not